aws/solutions/AmazonCloudWatchAgent/inline/amazon_linux.template

```diff
@@ -6,15 +6,25 @@ Parameters:
     Description: Name of an existing EC2 KeyPair to enable SSH access to the instance
     Type: AWS::EC2::KeyPair::KeyName
     ConstraintDescription: must be the name of an existing EC2 KeyPair.
+  PackageLocation:
+    Description: Location to download the agent from
+    Type: String
+    ConstraintDescription: must be the location of the package to download
+    Default: s3://amazoncloudwatch-agent/amazon_linux/amd64/latest/amazon-cloudwatch-agent.rpm
+  MetricNamespace:
+    Description: Metric Namespace
+    Type: String
+    ConstraintDescription: must be a string
+    Default: YourNameSpaceHere
```

Comment:
Should the default for the `MetricNamespace` variable be `CWAgent`.
Isn't that what happens today if you omit `"namespace"` from the CWA's JSON config?